src/confcom/azext_confcom/container.py

```diff
@@ -22,268 +22,305 @@
-                        + "can only be a boolean."
-                    )
+            exec_processes_output.append(
+                {
+                    config.POLICY_FIELD_CONTAINERS_ELEMENTS_COMMANDS: exec_command,
+                    config.POLICY_FIELD_CONTAINER_SIGNAL_CONTAINER_PROCESSES: exec_signals,
+                    config.POLICY_FIELD_CONTAINERS_ALLOW_STDIO_ACCESS: exec_stdio_access,
+                }
+            )
+    return exec_processes_output
 
-                exec_processes_output.append(
-                    {
-                        config.POLICY_FIELD_CONTAINERS_ELEMENTS_COMMANDS: exec_command,
-                        config.POLICY_FIELD_CONTAINER_SIGNAL_CONTAINER_PROCESSES: exec_signals,
-                        config.POLICY_FIELD_CONTAINERS_ALLOW_STDIO_ACCESS: exec_stdio_access,
-                    }
-                )
 
-        # get the signals info used as a liveness probe
-        signals = (
-            case_insensitive_dict_get(
-                container_json, config.ACI_FIELD_CONTAINERS_SIGNAL_CONTAINER_PROCESSES
+# change
+def extract_allow_elevated(container_json: Any):
+    _allow_elevated = case_insensitive_dict_get(
+        container_json, config.ACI_FIELD_CONTAINERS_ALLOW_ELEVATED
+    )
+    if _allow_elevated:
+        if not isinstance(_allow_elevated, bool):
+            eprint(
+                f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
+                + f'["{config.ACI_FIELD_CONTAINERS_ALLOW_ELEVATED}"] can only be boolean value.'
             )
-            or []
+    else:
+        # default is allow_elevated should be true
+        _allow_elevated = True
+    return _allow_elevated
+
+
+def extract_allow_studio_access(container_json: Any):
+    # get the field for Standard IO access, default to true
+    allow_stdio_value = case_insensitive_dict_get(
+        container_json, config.ACI_FIELD_CONTAINERS_ALLOW_STDIO_ACCESS
+    )
+    allow_stdio_access = allow_stdio_value if allow_stdio_value is not None else True
+    return allow_stdio_access
+
+
+def extract_get_signals(container_json: Any):
+    # get the signals info used as a liveness probe
+    signals = (
+        case_insensitive_dict_get(
+            container_json, config.ACI_FIELD_CONTAINERS_SIGNAL_CONTAINER_PROCESSES
         )
-        if signals:
-            if not isinstance(signals, list):
+        or []
+    )
+    if signals:
+        if not isinstance(signals, list):
+            eprint(
+                f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
+                + '["{config.ACI_FIELD_CONTAINERS_SIGNAL_CONTAINER_PROCESSES}"]'
+                + "can only be a list."
+            )
+
+        for signals_item in signals:
+            if not isinstance(signals_item, int):
                 eprint(
                     f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
                     + '["{config.ACI_FIELD_CONTAINERS_SIGNAL_CONTAINER_PROCESSES}"]'
```

Comment:
this should be an f-string

Reply:
I'm not sure what the issue is here. This is directly copied from the original code. Should we change the code to this? 
```
eprint(
                    f'Field ["{config.ACI_FIELD_CONTAINERS}"]'
                    + f'["{config.ACI_FIELD_CONTAINERS_SIGNAL_CONTAINER_PROCESSES}"]'
                    + f"can only be an integer."
                )
```

Reply:
mostly, only the first two lines need the "f"s. That must be a bug in the old code too